ouranos-config/proxy/catalog.conf.template

```diff
@@ -0,0 +1,6 @@
+    location /catalog {
```

Comment:
Forgot to add `catalog.conf` to `.gitignore` in this PR.  It's done in commit https://github.com/bird-house/birdhouse-deploy-ouranos/commit/1e885063666603748ce0277721f2f185fedd73ab